environment.yml

```diff
@@ -1,12 +1,7 @@
-name: simulate_expression_compendia
+name: test_ponyo
```

Comment:
Why is your environment called `test_ponyo` instead of `ponyo`?

Also if this package is installable with pip why have a `environment.yml` file instead of a `requirements.txt` file?